components/cloud-foundry/frontend/src/view/applications/application/summary/add-route/add-route.service.js

```diff
@@ -11,9 +11,11 @@
    * @memberof cloud-foundry.view.applications.application.summary
    * @param {app.model.modelManager} modelManager - the Model management service
    * @param {object} frameworkAsyncTaskDialog - async dialog service
+   * @param {object} $q - promise library
+   * @param {object} appClusterRoutesService - App cluster router service
    * @constructor
    */
-  function AddRouteServiceFactory(modelManager, frameworkAsyncTaskDialog) {
+  function AddRouteServiceFactory(modelManager, frameworkAsyncTaskDialog, $q, appClusterRoutesService) {
```

Comment:
As a general rule we try to put angular $ services first. This practise wasn't applied from start though so you may still find places where they appear anywhere